SU-SWS/decanter
# V6 d8core-2832 focus for links

# Not READY FOR Review!


# Summary
Establish a new new focus to be used across all focusable items.

# Needed By (Date)
v 2.0.1

# Urgency
Not

# Steps to Test

1. Check out this branch
2. Develop sufficient examples of focusable items
3. Verify that the new focus state works on focusable items.

![screenshot-focus-2021 01 05-18_34_29](https://user-images.githubusercontent.com/284440/103722496-f1722880-4f84-11eb-843a-4dd56acc4c6e.png)


# Affected Projects or Products
- D8Core and related sites
- Anything using Decanter

# Associated Issues and/or People
- JIRA ticket D8CORE-2832
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@ishahism 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
